products/devex/README.md

```diff
@@ -57,3 +57,170 @@ An example is given below
   ]
 }
 ```
+
```

Comment:
FWIW, in the app specific documentation I will keep only the information relevant to the app within the app specific informations and point the people to the central doc for the general ones.

Reply:
READMEs has been updated. Thanks!